packages/features/src/categories/error-tracking/patterns/rollbar.ts

```diff
@@ -0,0 +1,128 @@
+import { Page } from 'playwright';
+
+export const rollbar = [
+  {
+    name: 'globals' as const,
+    score: 1.5,
+    browser: async (page: Page) => {
+      return page.evaluate(() => {
+        const markers = {
+          hasRollbar: !!window.Rollbar,
+          hasRollbarConfig: !!window._rollbarConfig,
+          hasRollbarAccessToken:
+            typeof window._rollbarAccessToken !== 'undefined',
+          hasRollbarShims: !!window._rollbarShims,
+          hasRollbarInitialized: !!window._rollbarInitialized,
+          hasRollbarWrappedError: !!window._rollbarWrappedError,
+          hasRollbarDidLoad: typeof window._rollbarDidLoad !== 'undefined',
+          hasRollbarOnError:
+            window.onerror &&
+            window.onerror.toString().includes('_rollbarWrappedError'),
+        };
+        return Object.values(markers).some(Boolean);
```

Comment:
Note, that in browser, we return only boolean. It means that doesn't metter if 1 or all matches happened. So this category will add only 1.5 score. Maybe we can split it to few groups (globals, shims, errors) to increase confidence.